rmgpy/kinetics/diffusionLimitedTest.py

```diff
@@ -111,6 +139,14 @@ def testGetEffectiveRate2to1(self):
         self.assertTrue(effective_rate < self.intrinsic_rates[self.bi_uni_reaction])
         self.assertTrue(effective_rate >= 0.2 * self.intrinsic_rates[self.bi_uni_reaction])
 
+    def testGetEffectiveRate3to2(self):
+        """
+        Tests that the effective rate is limited for a 3 -> 2 reaction
+        """
+        effective_rate = diffusionLimiter.getEffectiveRate(self.tri_bi_reaction, self.T)
+        self.assertTrue(effective_rate < self.intrinsic_rates[self.tri_bi_reaction])
+        self.assertTrue(effective_rate >= 0.2 * self.intrinsic_rates[self.tri_bi_reaction])
```

Comment:
Where does the 0.2 come from?

Reply:
I think it's basically just for checking if the new rate was reduced too much by the diffusion limit, which would be very unlikely. I used 0.2 because that's what was used in `testGetEffectiveRate2to1`.